exercises/say/uSayExample.pas

```diff
@@ -0,0 +1,77 @@
+unit uSay;
+
+interface
+
+type
+  TSay = class
+    class function Say(AInp : Int64) : string;
+  end;
```

Comment:
Let's change it up a little.  Just a function this time.  There is no need for a class.
Just this in the interface:
```pascal
function Say(AInp: Int64): string;
```
Saying (pardon the pun) TSay.Say is a little awkward.